Microsoft.Toolkit.Uwp.UI.Controls/Menu/Menu.cs

```diff
@@ -57,6 +57,25 @@ public Orientation Orientation
                 new PropertyMetadata(Orientation.Horizontal, OrientationPropertyChanged));
 
         /// <summary>
+        /// Gets or sets the menu flyout placement
+        /// </summary>
+        public MenuFlyoutPlacement FlyoutPlacement
```

Comment:
What if, what if, what if... Menus have a predefined behavior. That's what should be supported. Even allowing a vertical orientation is a stretch.